packages/sdk/src/index.ts

```diff
@@ -93,42 +94,48 @@ export class Gomu {
   }
 
   async makeOrder(params: MakeOrderParams): Promise<Order[]> {
-    // @ts-ignore
     return Promise.all(
       Object.entries(this.marketplaces)
-        .filter(([_, marketplace]) => marketplace)
-        .map(async ([marketplaceName, marketplace]) => ({
-          marketplaceName,
-          marketplaceOrder: await marketplace.makeOrder(params),
-        }))
+        .filter(([marketplaceName, marketplace]) => {
+          if (
+            params.marketplaces?.length &&
+            !params.marketplaces.includes(marketplaceName as MarketplaceName)
+          ) {
+            return false;
+          }
+
+          return Boolean(marketplace);
+        })
+        .map(async ([marketplaceName, marketplace]) => {
+          return {
+            marketplaceName,
+            marketplaceOrder: await marketplace.makeOrder(params),
+          } as Order;
```

Comment:
Allows us to remove @ts-ignore. That's why I needed to remove the arrow rule, with it I couldn't do this return without creating order variable first.

Reply:
Not sure why typescript can't recognise this as well. `as Order` is kinda the same as `@ts-ignore` here. but I guess `as Order` is "cleaner"

It's complaining because it can't guarantee that opensea marketplaceName will return an opensea marketOrder, somehow.